.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
```

Comment:
Having this file in your repository will enable the GitHub-native Dependabot.

After you merge this PR, Dependabot will run right away, and create updates for your GitHub Action dependencies, and for your npm dependencies.